ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10.yaml

```diff
@@ -11,6 +11,10 @@ base_images:
     name: "4.10"
     namespace: ocp
     tag: tools
+  upi-installer:
```

Comment:
I added upi-installer as it is needed by the ipi-conf-aws-windows-machineset ref and it is included on the other config files for 4.11, 4.12, 4.13 and master.